lib/rules/no-unused-vars.js

```diff
@@ -126,6 +131,29 @@ module.exports = {
         }
 
         /**
+         * Determines if a variable has a sibling rest property
+         * @param {Variable} variable - EScope variable object.
+         * @returns {boolean} True if the variable is exported, false if not.
+         * @private
+         */
+        function hasRestSpreadSibling(variable) {
+            if (config.ignoreRestSiblings === true) {
```

Comment:
Nitpick: This can be replaced with `if (config.ignoreRestSiblings) {` (explicitly comparing to `true` isn't necessary).